out/artifacts/web_intro_war_exploded/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,5 @@
+Manifest-Version: 1.0
+Created-By: IntelliJ IDEA
+Built-By: Nikita
+Build-Jdk: version 15.0.2
+
```

Comment:
please remove all the files in `out` folder from the PR
they should not be pushed to Github
`out` folder is another name for `target` folder - it should be excluded in `.gitignore` if you are building the project into the `out` folder